TelCo.ColorCoder/ManualMap.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+  class ManualMap{
+    
+    public static void ToString(){
```

Comment:
Resolve this compiler warning. Do you understand what it means?

Reply:
Yes. I used ToString as method name and that hides the functionality of in-built ToString method.

Reply:
ok add override
Also for all the comments, move them to resolved once done